deployment/postgres-init-db/sql/tables/scheduler/scheduling_goal_definition.sql

```diff
@@ -2,7 +2,10 @@ create table scheduler.scheduling_goal_definition(
   goal_id integer not null,
   revision integer not null default 0,
 
-  definition text not null,
+  type scheduler.goal_type not null default 'EDSL',
+  definition text,
+  uploaded_jar_id integer,
+  parameter_schema jsonb,
```

Comment:
To confirm for my understanding: Are EDSL goals allowed to have`parameter_schema`s?

Reply:
If we choose to continue support for the EDSL, yes, I think we would want to extend them to allow for parameters. If procedural scheduling goes well, we may end up deprecating the EDSL instead of putting further development work into it.